### DISCUSSION TITLE
IOS splash screen

### DISCUSSION THREAD

**Discussion by minghwee** [timbru31/cordova-plugin-lottie-splashscreen#456] [Q&A] [answered]
Hi

Would like to check if there is any workaround for LottieScaleType if we want to use in IOS?
I currently have LottieScaleType with value CENTER on my android app and would like to have the same behaviour for the IOS app.
Thank you

**Comment by timbru31** [comment, marked as answer]
That should be the default. Can you show a screenshot of the (misplaced) animation on iOS?
